source/inverse_model.py

```diff
@@ -79,6 +81,14 @@ def __init__(self, flownetwork: flow_network.FlowNetwork, imp_readtargetvalues:
         self._imp_adjointmethodsolver = imp_adjointmethodsolver
         self._imp_alphamapping = imp_alphamapping
 
+        # Solution monitoring - Visualisation
+        self._current_iteration = 0
+        self._iteration_array = np.array([], dtype=np.int)
```

Comment:
generally, i use the underline "_" for variables if they are not accessed from outside. however, in your implementation you access them from "solution_monitoring_inverseproblem.py". therefore, they are not internal functions of inverse_model.py and should be without an underline "_"

Reply:
super